{{ cookiecutter.hosting_github_repo_name }}/README.md

```diff
@@ -0,0 +1,141 @@
+
+Council Data Project is an open-source project dedicated to providing journalists,
+activists, researchers, and all members of each community we serve with the tools they
+need to stay informed and hold their Council Members accountable.
+
+For more information about CouncilDataProject, please visit
+[our website](https://councildataproject.github.io/).
+
+## Instance Information
+
+This repo serves the municipality: **{{ cookiecutter.municipality }}**
+
+## Contributing
+
+If you wish to contribute to CDP please note that the best method to do so is to
+contribute to the upstream libraries that compose the CDP Instances themselves.
+These are detailed below.
+
+-   [cdp-backend](https://github.com/CouncilDataProject/cdp-backend): Contains
+    all the database models, data processing pipelines, and infrastructure-as-code for CDP
+    deployments. Contributions here will be available to all CDP Instances. Entirely
+    written in Python.
+-   [cdp-frontend](https://github.com/CouncilDataProject/cdp-frontend): Contains all of
+    the components used by the web apps to be hosted on GitHub Pages. Contributions here
+    will be available to all CDP Instances. Entirely written in
+    TypeScript and React.
+-   [cookiecutter-cdp-deployment](https://github.com/CouncilDataProject/cookiecutter-cdp-deployment):
+    The repo used to generate new CDP Instance deployments. Like this repo!
+-   [councildataproject.github.io](https://github.com/CouncilDataProject/councildataproject.github.io):
+    Our landing page! Contributions here should largely be text changes and admin updates.
+
+## Initial Repo Setup
+
+1.  Create (or sign in to) a Google Cloud Platform (GCP) account.
+    ([Google Cloud Console Home](https://console.cloud.google.com/))<br>
+    Google Cloud Platform is where all data and files will be stored, and some
+    processing will be done using GCP resources.
+    More details available in the [Google Cloud](#google-cloud) section.
+2.  Create (or re-use) a [billing account](https://console.cloud.google.com/billing)
+    and attach it to your GCP account.<br>
+    For more details on the cost of maintaining a CDP Instance, see [Cost](#cost).
+3.  Create a new [Google Cloud Project](https://console.cloud.google.com/projectcreate).
+    -   Set the project name to: **cdp-{{ cookiecutter.municipality_slug }}**.
+4.  After clicking "Create", Google Cloud will take a second to prevision the project.
+    Once it has, you will have access to the
+    [project's dashboard](https://console.cloud.google.com/home/dashboard?project=cdp-{{ cookiecutter.municipality_slug }}).
+5.  Create a [Service Account](https://console.cloud.google.com/iam-admin/serviceaccounts/create?project=cdp-{{ cookiecutter.municipality_slug }}).
+    -   Set the Service Acount Name to: "GitHub Actions Runner".
+    -   Click "Create".
+    -   Grant the access to the project for the service account:
+        -   Select the "Owner" (Full access to all resources) role.
```

Comment:
Might be worth a short sentence saying why 'Owner` is needed here; full access seems so broad (but perhaps needed in this case?)

Reply:
It's a good question. I think it could be lowered to Editor which is a tiny bit safer but still pretty broad. This is a game of how easy to we want it to be to deploy. AND, GCP doesn't have as nearly as nice an IAM + Credential system as AWS for example.

I will change to Editor but could easily be changed later.